web/src/test/java/no/nav/foreldrepenger/web/app/tjenester/fpoversikt/InntektsmeldingDtoTjenesteTest.java

```diff
@@ -22,41 +49,155 @@
 import no.nav.foreldrepenger.domene.typer.Beløp;
 import no.nav.foreldrepenger.domene.typer.JournalpostId;
 
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.junit.jupiter.MockitoExtension;
+
 @ExtendWith(JpaExtension.class)
+@ExtendWith(MockitoExtension.class)
```

Comment:
Trenger du både JpaExtension og MockitoExtension? Bør det ikke holde med mockito?

Reply:
Tror jeg trenger JpaExtension for EntityManager bruk